### PR TITLE
adding klaytn token standards

### PR DESCRIPTION
**Pull request by 0xxlegolas**
## Proposed changes

This PR is for adding klaytn token standards to the master branch of openzepplin contracts commit id [6339027](https://github.com/klaytn/klaytn-contracts/tree/6339027a7ae783dcb462f6bd6a5be189e46ea449)

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or Enhancement
- [ ] Others

## Comments 
klaytn contract library to support KIP7, KIP17 and KIP37 contracts

### CODE REVIEW

**Review by chloe-klatyn — APPROVED** [review]

**Review by baerae-colin — APPROVED** [review]
LGTM

**Review by terry-krust — APPROVED** [review]

**Review by terry-klaytn — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+#*       @global-owner1 @global-owner2
+*       @terry-klaytn @neoofklaytn @toniya-klaytn @chloe-klatyn
```
Comment (neoyiu0x) [inline .github/CODEOWNERS]: the codeowner should be updated and could consider including some of our core dev leads thanks

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,64 +1,76 @@
...
 
-*IMPORTANT* Read the PR template very carefully and make sure to follow all the instructions. These instructions
-refer to some very important conditions that your PR must meet in order to be accepted, such as making sure that all tests pass, JS linting tests pass, Solidity linting tests pass, etc.
+You can use the following labels:
 
-6) Maintainers will review your code and possibly ask for changes before your code is pulled in to the main repository. We'll check that all tests pass, review the coding style, and check for general code correctness. If everything is OK, we'll merge your pull request and your code will be part of OpenZeppelin Contracts.
+Labels for initial issue categories:
 
-*IMPORTANT* Please pay attention to the maintainer's feedback, since it's a necessary step to keep up with the standards OpenZeppelin Contracts attains to.
+- issue/bug: Issues with the code-level bugs.
```
Comment (neoyiu0x) [inline CONTRIBUTING.md:115]: Issues types should reflect:
[Issues and Bugs]
[Feature Requests]
[Code Contribution]

as we listed above thanks

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,64 +1,76 @@
...
+8. After merging the PR, the pull request will be closed. You can then delete the now obsolete branch.
 
-## Creating Pull Requests (PRs)
+## Types of Contribution
+There are various ways to contribute and participate. Please read the guidelines below regarding the process of each type of contribution.
 
-As a contributor, you are expected to fork this repository, work on your own fork and then submit pull requests. The pull requests will be reviewed and eventually merged into the main repo. See ["Fork-a-Repo"](https://help.github.com/articles/fork-a-repo/) for how this works.
+-   [Issues and Bugs](#issues-and-bugs)
+-   [Feature Requests](#feature-requests)
+-   [Code Contribution](#code-contribution)
```
Comment (neoyiu0x) [inline CONTRIBUTING.md:21]: content of #code-contribution will need to be created somewhere in this markdown

**Review by neoofkaia — COMMENTED** [review]
There are only few minor ones I left here and it should be good to go thanks @toniya-klaytn

File: .github/CODEOWNERS (hunk 1 above)
Comment (0xxlegolas) [reply .github/CODEOWNERS]: added and pushed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (0xxlegolas) [reply CONTRIBUTING.md:115]: refactored

File: CONTRIBUTING.md (hunk 3 above)
Comment (0xxlegolas) [reply CONTRIBUTING.md:21]: added and pushed the code

**Review by neoofkaia — APPROVED** [review]

**Review by aidan-kwon — APPROVED** [review]

**Pull request merged**